### PR TITLE
Implement bytes.splitlines()

### PR DESCRIPTION
**Pull request by wkeeling**
Refs: #38 

Implementation of `bytes.splitlines()` with tests.

Also fixes a couple of syntax problems discovered in `microbenchmarks,py`

### CODE REVIEW

File: tests/microbenchmarks.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from os import path
 sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
 
-from utils import *
+from .utils import *
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E402) module level import not at top of file

Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (F403) 'from .utils import *' used; unable to detect undefined names

File: tests/microbenchmarks.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
 def test_small_integers(test_case):
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E302) expected 2 blank lines, found 1

File: tests/microbenchmarks.py [hunk 3]
```diff
@@ -203,7 +203,7 @@ def test_dict_set(test_case):
             dict["meow"] = 4
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """), timed=True)
 
 def test_class_init(test_case):
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E302) expected 2 blank lines, found 1

File: tests/microbenchmarks.py [hunk 4]
```diff
@@ -203,7 +203,7 @@ def test_dict_set(test_case):
...
     print("Running" , "test_class_init")
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 20: (E203) whitespace before ','

**Comment by phildini** [comment]
Hi there! It looks like this PR might be dead, so we're closing it for now. Feel free to re-open it if you'd like to continue, or think about directing your efforts to https://github.com/beeware/briefcase or https://github.com/beeware/toga. Both of these have more active development right now. 😄

**Pull request closed without merging**